packages/netsuite-adapter/src/client/sdf_client.ts

```diff
@@ -106,9 +106,11 @@ const FEATURE_ID = 'featureId'
 const configureFeatureFailRegex = RegExp(`Configure feature -- (Enabling|Disabling) of the (?<${FEATURE_ID}>\\w+)\\(.*?\\) feature has FAILED`)
 
 export const OBJECT_ID = 'objectId'
+const FEATURE_NAME = 'featureName'
 const deployStartMessageRegex = RegExp('^Begin deployment$', 'm')
 const settingsValidationErrorRegex = RegExp('^Validation of account settings failed.$', 'm')
 export const objectValidationErrorRegex = RegExp(`^An error occurred during custom object validation. \\((?<${OBJECT_ID}>[a-z0-9_]+)\\)`, 'gm')
+const objectValidationFeatureErrorRegex = RegExp(`Details: You must specify the (?<${FEATURE_NAME}>[A-Z_]+)\\([a-zA-Z ]+\\) feature in the project manifest`, 'gm')
```

Comment:
I'm not sure if feature names can be with numbers/special characters, so maybe this is more inclusive-
```suggestion
const objectValidationFeatureErrorRegex = RegExp(`Details: You must specify the (?<${FEATURE_NAME}>[A-Z_]+)\\(.*?\\) feature in the project manifest`, 'gm')
```